lib/db/schema/patch-039-040.sql

```diff
@@ -0,0 +1,66 @@
+ALTER TABLE `verificationReminders` DROP PRIMARY KEY, ADD PRIMARY KEY (`uid`, `type`);
+
+CREATE PROCEDURE `fetchVerificationReminders_2` (
+    IN currentTime BIGINT SIGNED,
+    IN reminderType VARCHAR(255),
+    IN reminderTime BIGINT SIGNED,
+    IN reminderTimeOutdated BIGINT SIGNED,
+    IN reminderLimit INTEGER
+)
+BEGIN
+  DECLARE EXIT HANDLER FOR SQLEXCEPTION
+  BEGIN
+    DO RELEASE_LOCK('fxa-auth-server.verification-reminders-lock');
+    ROLLBACK;
+    RESIGNAL;
+  END;
+
+  START TRANSACTION;
+
+  DO @lockAcquired:=GET_LOCK('fxa-auth-server.verification-reminders-lock', 1) AS lockAcquired;
```

Comment:
Do we need the `AS lockAcquired` part if we're assigning to varialbe with `@lockAcquired:=`?  I don't recall the precise details of the semantics here, will try to poke at this a little tomorrow.